tool/jt.rb

```diff
@@ -756,9 +765,31 @@ def test_mri(*args)
                       else
                         []
                       end
-
       files_to_run = (include_files - exclude_files)
+      
+    end
+
+    cext_tests = files_to_run.select { |f| f.include?("cext/ruby") }
+    cext_tests.each do |test|
+      test_path = "#{TRUFFLERUBY_DIR}/test/mri/tests/#{test}"
+      match = File.open(test_path).read().match(/^require ['"]c\/(.*?)["']/)
+      if match
+        compile_dir = if match[1].include?('/')
+                        if Dir.exists?("#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c/#{match[1]}")
+                           "#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c/#{match[1]}"
+                        else
+                          parent_dir = match[1].split('/').slice(0..-2).join('/')
+                          "#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c/#{parent_dir}"
+                        end
+                      else
+                        "#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c/#{match[1]}"
```

Comment:
Can you extract `"#{TRUFFLERUBY_DIR}/test/mri/tests/cext/c` in a local or constant?